packages/core/src/components/widgets/RadioWidget.tsx

```diff
@@ -0,0 +1,64 @@
+import React from "react";
+import { WidgetProps } from "@rjsf/utils";
+
+function RadioWidget<T = any, F = any>({
```

Comment:
Can you add JSDoc here as well... take a look at how the other, already converted widgets are documented as a guide